docs/quickstarts/nextjs.mdx

```diff
@@ -52,7 +52,7 @@ In your Next.js project's root folder, you may have an `.env.local` file alongsi
 
 Add the following code to your `.env.local` file to set your public and secret keys.
 
-**Pro tip!** If you are signed into your [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=api-keys), your secret key should become visible by clicking on the eye icon.
+**Pro tip!** If you are signed into your Clerk Dashboard, your secret key should become visible by clicking on the eye icon. Otherwise, you can find your keys in the Clerk Dashboard on the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page.

```

Comment:
I've changed this line and a bunch of other stuff in the next.js quickstart, so it's probably best to drop this change to avoid a merge conflict

https://github.com/clerk/clerk-docs/pull/703